internal/systemd/systemd.go

```diff
@@ -48,3 +50,38 @@ func WaitOnDevices(devs []string, stage string) error {
 
 	return nil
 }
+
+// GetSystemdVersion fetches the version of Systemd
+// in a given system.
+func GetSystemdVersion() (uint, error) {
+	var systemdVersion string
+	var begin, end int
+	conn, err := dbus.NewSystemConnection()
+	if err != nil {
+		return 0, err
+	}
+	version, err := conn.GetManagerProperty("Version")
+	if err != nil {
+		return 0, err
+	}
+	// Handle different systemd versioning schemes that are being returned.
+	// for e.g:
+	// - Fedora 31: `v243.5-1.fc31`
+	// - RHEL 8: `239`
+	if strings.HasPrefix(version, "v") {
+		begin = 1
+
```

Comment:
extra (not needed) blank line right here ^^

Reply:
Fixed.